doc/api/cli.md

```diff
@@ -340,6 +358,16 @@ misformatted, but any errors are otherwise ignored.
 Note that neither the well known nor extra certificates are used when the `ca`
 options property is explicitly specified for a TLS or HTTPS client or server.
 
+### `SSL_CERT_DIR=dir`
+
+If `--use-openssl-ca` is enabled, this overrides and sets OpenSSL's directory
+containing trusted certificates.
+
+### `SSL_CERT_FILE=file`
```

Comment:
This is better and it's good enough for me to land. 

@AdamMajer ... as @sam-github mentions, the issue is about making sure the average user knows what the impact of the options are, even if they are not already familiar with OpenSSL.